src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,189 @@
-import React, { Component } from 'react';
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 
-export class NewMovie extends Component {
-  state = {};
+import './NewMovie.scss';
+
+export class NewMovie extends React.Component {
+  state = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+
+    errors: {
+      title: '',
+      imgUrl: '',
+      imdbUrl: '',
+      imdbId: '',
+    },
+  };
+
+  inputChange = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({ [name]: value });
+  }
+
+  inputValidation = (event) => {
```

Comment:
```suggestion
  validateInput = (event) => {
```

when naming function, use VERBS